content/app/development/configuration/queryparameters/_index.en.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Query parameters
+description: Available query parameter for an app.
+weight: 200
+tags: [translate-to-english]
+---
+
+## returnUrl
+
+Query parameter `returnUrl` is used by the app to redirect users to that value when they click the exit-button in the app.
+
+Valid values for `returnUrl` is value with a valid URL-format and that the host name defined in the value is the same one
```

Comment:
This is a word heavy sentence.. Could we rephrase it somehow? 
Somthing about the parameter must be formated as a valid URL, and the url must be within the altinn-domain ?

Reply:
Then we should list all the relevant domains in my opinion, I dont know all of them, I know of tt02, altinn.no, altinn3local.no. But if you are happy with just saying "within the altinn-domain", Ill change to that :)

Reply:
Also, one can not point to a different host inside the altinn-domain. One must point to the same host. I.e: tt02.altinn.no URL must be pointed to tt02.altinn.no, it can not point to altinn.no or altinn3local.no,

Reply:
tt02, and altinn.no should be enough infor for external user's these are the only environments  made available to them. Altinn3local is going to change soon, so maybe leave it out and we add it to the documentation of localtest instead